### ISSUE TITLE
Update Link color across the CMS to meet WCAG 2.0 AAA contrast requirement

### ISSUE THREAD

**Issue by cjterdi** [department-of-veterans-affairs/va.gov-cms#10046]
## Description
Reference design tickets:

[9916](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/9916): The standard link style is blue underline with a hover style of a light blue background and no underline. This combination of blue does not pass the WCAG 2.0 AAA contrast requirements.

[9917](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/9917): An accessibility audit found that on all products and all tables, the main content listing has tables that do not pass WCAG accessibility standards for hover states. Table rows currently have a hover state that changes to light blue.

[9915](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/9915): On staging, the footer links have a background color of light blue which introduces an accessibility issue.

See approved [link design](https://www.figma.com/file/ScABI31f3MRFEvQ8WitVPa/CMS-Design-System?node-id=1956%3A95319)

The following changes to the links are documented here: [CMS DS Changelog ](https://www.figma.com/file/ktDP1ISywa2Lc5q3GC8ezh/Changelog?node-id=2%3A38)

- All standard links will be changed to the dark blue (004795)
- hover/focus state link color will be darker blue (003e73)
- We think having a background color would be beneficial as this would keep it as a similar format as the front end (they have a background color on hover) This is not an accessibility requirement.
- The hover/focus state should be the lighter blue (e4eff9). This is currently the color on some CMS links, not all. It is also different than the VA.gov front end hover color
- Footer links will remain white
- Footer links on hover/focus will keep their current background color (e4eff9) but the text will be darker blue (003e73)
- All links should have underline on default and then no underline on hover/focus

There is an additional condition for links in read-only content. To meet the same requirements we made an adjustment to the grey lighter color to #E5E3E1. See changes are documented here: [CMS DS Changelog](https://www.figma.com/file/ktDP1ISywa2Lc5q3GC8ezh/Changelog?node-id=2%3A2) 

## Acceptance Criteria
- [ ] Implement the design changes listed above across CMS
- [ ] Design and accessibility review
- [ ] QA

### CMS Team
Please check the team(s) that will do this work.

- [ ] `Program`
- [ ] `Platform CMS Team`
- [ ] `Sitewide Crew`
- [x] `⭐️ Sitewide CMS`
- [ ] `⭐️ Public Websites`
- [ ] `⭐️ Facilities`
- [ ] `⭐️ User support`

**Comment by EWashb** [comment]
As Edmund ramps up to the project, this may need to stretch. He is becoming familiar with the code base now and this is a good first task for him to do that.

**Comment by EWashb** [comment]
This ticket (the implementation) will stretch into sprint 65. however, this was not planned for sprint 64 specifically, just the work began as new dev was being onboarded.

**Comment by EWashb** [comment]
This could possibly be blocked for sprint 65 as Edmund is out sick

**Comment by laflannery** [comment]
This needs to be reopened after I was notified of some issues with some of the links.  These are issues that Edmund and I had worked through and resolved previously.  Examples are (but these are not all of them):
- Edit buttons within the Content Listing tables are now white - they should have remanined the color they were previously (I believe it was `#212121`)
- The active tab text is white
- The Content dropdown text is white
- Tabs have underlines

Screenshot of the first 3
![image](https://user-images.githubusercontent.com/106678594/188024222-9c25e5d4-5d72-4ee2-8ee9-26ab20315b77.png)

Screenshot of the underlined tabs
![image](https://user-images.githubusercontent.com/106678594/188024191-293698e1-d287-44d3-9333-83d5e410b01c.png)

I also happened to see all these issues when I was reviewing the Admin toolbar PR - I feel like it's not related to why this is all appearing now but I also felt like I needed to say it just in case

**Comment by EWashb** [comment]
@edmund-dunn fixing this will need to take priority over all other work this sprint.

**Comment by laflannery** [comment]
I think the below updates should fix the issues:

- Add `.layout-container a.tabs__link {
    text-decoration: none;
}`
This is needed to remove underlines from tabs

- Replace `.layout-container a.button, .layout-container a.is-active, .layout-container a.feed-icon` with `.layout-container a.button, .layout-container a.pager__link.is-active, .layout-container a.feed-icon`
This will target only the active paging link instead of all active links

- Replace `table .button, table .button:not(:focus)` with `table a.button, table a.button:not(:focus)`
  - AND add the property color: var(--button-fg-color);
This will change the color of the table buttons back to the standard text color

I do think this will resolve the issues we are seeing and I don't think it will cause other interactions but obviously needs to be retested

**Comment by laflannery** [comment]
@edmund-dunn If you have a min can you take a look at this?  This was the issue that got a bit confused and I think got reverted but then maybe we never went back and redid it.  I'm not seeing your changes on Prod

**Comment by dawnpruitt** [comment]
@JayDarnell please reach out to @laflannery; this ticket isn't as clear cut as it first appears and she has the context you'll need. Thank you!

**Comment by edmund-dunn** [comment]
This was reverted because of regressions.
